### ISSUE TITLE
Unable to start the program

### ISSUE THREAD

**Issue by wayfar3r** [fuzzbunch/fuzzbunch#1]
Actually, tested this on several x64 systems (Win 10, Win 7) and all of them give the same error:

`Traceback (most recent call last):
  File "C:\Users\Dmitry\Desktop\fuzzbunch-master\fb.py", line 37, in <module>
    from fuzzbunch.edfplugin import EDFPlugin
  File "C:\Users\Dmitry\Desktop\fuzzbunch-master\fuzzbunch\edfplugin.py", line 7, in <module>
    from plugin       import Plugin
  File "C:\Users\Dmitry\Desktop\fuzzbunch-master\fuzzbunch\plugin.py", line 8, in <module>
    import truantchild
  File "C:\Users\Dmitry\Desktop\fuzzbunch-master\fuzzbunch\truantchild.py", line 8, in <module>
    import exma
  File "C:\Users\Dmitry\Desktop\fuzzbunch-master\fuzzbunch\exma.py", line 17, in <module>
    _libraries['exma.dll'] = ctypes.CDLL('exma-1.dll')
  File "C:\Program Files\Python\lib\ctypes\__init__.py", line 362, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 193] %1 is not a valid Win32 application`

After installing 32-bit version of Python and pywin32, the error changes to this:

`Traceback (most recent call last):
  File "C:\Users\Noname\Desktop\fuzzbunch-master\fb.py", line 37, in <module>
    from fuzzbunch.edfplugin import EDFPlugin
  File "C:\Users\Noname\Desktop\fuzzbunch-master\fuzzbunch\edfplugin.py", line 7, in <module>
    from plugin       import Plugin
  File "C:\Users\Noname\Desktop\fuzzbunch-master\fuzzbunch\plugin.py", line 8, in <module>
    import truantchild
  File "C:\Users\Noname\Desktop\fuzzbunch-master\fuzzbunch\truantchild.py", line 12, in <module>
    import pytrch as trch
  File "C:\Users\Noname\Desktop\fuzzbunch-master\lib\x86-Windows\pytrch.py", line 25, in <module>
    _pytrch = swig_import_helper()
  File "C:\Users\Noname\Desktop\fuzzbunch-master\lib\x86-Windows\pytrch.py", line 21, in swig_import_helper
    _mod = imp.load_module('_pytrch', fp, pathname, description)
ImportError: DLL load failed: The specified module could not be found.`

**Comment by ryan-blanchard** [comment]
same issue
`Traceback (most recent call last):
  File "C:\Users\ME\Desktop\shadowbroker-master\windows\fb.py", line 37, in <module>
    from fuzzbunch.edfplugin import EDFPlugin
  File "C:\Users\ME\Desktop\shadowbroker-master\windows\fuzzbunch\edfplugin.py", line 7, in <module>
    from plugin       import Plugin
  File "C:\Users\ME\Desktop\shadowbroker-master\windows\fuzzbunch\plugin.py", line 8, in <module>
    import truantchild
  File "C:\Users\ME\Desktop\shadowbroker-master\windows\fuzzbunch\truantchild.py", line 8, in <module>
    import exma
  File "C:\Users\ME\Desktop\shadowbroker-master\windows\fuzzbunch\exma.py", line 17, in <module>
    _libraries['exma.dll'] = ctypes.CDLL('exma-1.dll')
  File "C:\Python26\lib\ctypes\__init__.py", line 353, in __init__
    self._handle = _dlopen(self._name, mode)
WindowsError: [Error 126] The specified module could not be found`

**Comment by 3xploit-hunt** [comment]
Basically same issue.

Info: 
Windows 10 (x64)
Python 2.7.12 (x32)
pywin32 (x32)

```
C:\Users\MyUsername\Desktop\shadowbroker\fuzzbunch-master>py -2 fb.py


Find module found: <open file 'C:\\Users\\MyUsername\\Desktop\\shadowbroker\\fuzzbunch-master\\lib\\x86-Windows\\_pytrch.pyd', mode 'rb' at 0x0350B5A0>


Traceback (most recent call last):
  File "fb.py", line 37, in <module>
    from fuzzbunch.edfplugin import EDFPlugin
  File "C:\Users\MyUsername\Desktop\shadowbroker\fuzzbunch-master\fuzzbunch\edfplugin.py", line 7, in <module>
    from plugin       import Plugin
  File "C:\Users\MyUsername\Desktop\shadowbroker\fuzzbunch-master\fuzzbunch\plugin.py", line 8, in <module>
    import truantchild
  File "C:\Users\MyUsername\Desktop\shadowbroker\fuzzbunch-master\fuzzbunch\truantchild.py", line 12, in <module>
    import pytrch as trch
  File "C:\Users\MyUsername\Desktop\shadowbroker\fuzzbunch-master\lib\x86-Windows\pytrch.py", line 28, in <module>
    _pytrch = swig_import_helper()
  File "C:\Users\MyUsername\Desktop\shadowbroker\fuzzbunch-master\lib\x86-Windows\pytrch.py", line 23, in swig_import_helper
    _mod = imp.load_module('_pytrch', fp, pathname, description)
ImportError: DLL load failed: The specified module could not be found.
```
I edited pytrch.py to print "Find module found...".

**Comment by cyberterror** [comment]
You need to use python 2.6 and pywin32.

**Comment by cyberterror** [comment]
Both x86

**Comment by wayfar3r** [comment]
Problem solved by using Python 2.6.
Thanks to @cyberterror
